aliases/available/git.aliases.bash

```diff
@@ -65,6 +65,7 @@ alias gco='git checkout'
 alias gcom='git checkout master'
 alias gcb='git checkout -b'
 alias gcob='git checkout -b'
+alias gcobu="git checkout -b ${USER}/"
```

Comment:
Please use single quote instead of double quotes to match the rest of the file

Reply:
I want to, but ${USER} won't expand if it's single quotes

Reply:
Hmm,  @georgettica how have you tested this?

_in my local_:
```
$ alias test='echo ${USER}/'
$ test

davidfa/

$ alias test

alias test='echo ${USER}/'

$ export USER=boo
$ test

boo/
```

This is in bash 5.0.7 on MacOS

Reply:
maybe it's me being silly. thanks so much @davidpfarrell .. seems it works and I was shooting from the hip. fixing now